test/props/mock/c.json

```diff
@@ -8,7 +8,11 @@
   },
 
   "aliases": {
-    "land": "green",
+    "baz": {
+      "value": "green",
+      "otherInfo": "ignoreMe"
+    },
+    "land": "{!baz}",
```

Comment:
I'd like to keep a `"key": "value"` pair somewhere, to make sure we're covering the most common use case in tests.